packages/api/src/routes/medical/document.ts

```diff
@@ -42,6 +44,23 @@ const getDocSchema = z.object({
   output: z.enum(["fhir", "dto"]).nullish(),
 });
 
+const searchSummarySchema = z.object({
+  question: z.string().min(3),
+});
+
+router.post(
```

Comment:
Should we move this to the patient's routes, since this is not really document related?

Maybe under `GET /patient/:id/consolidated/summary` to match the sibling "/search" endpoint?

Add TSDoc